Resources/config/form-type.xml

```diff
@@ -8,7 +8,7 @@
     </parameters>
 
     <services>
-        <service id="cmf_core.form.type.checkbox_url_label" class="%cmf_core.form.type.checkbox_url_label.class%">
+        <service id="cmf_core.form.type.checkbox_url_label" class="%cmf_core.form.type.checkbox_url_label.class%" public="false">
```

Comment:
in such case, you refer to it by its name. It is already loaded into the service in that case.